sdk/identity/identity/review/identity.api.md

```diff
@@ -172,6 +184,9 @@ export function getDefaultAzureCredential(): TokenCredential;
 
 export { GetTokenOptions }
 
+// @public
+export type IdentityExtension = (context: unknown) => void;
```

Comment:
since this is a public type, could the extensions import it and use it instead of exporting their own version?

Reply:
Yeah, this was just down to how I was importing it. Fixed.